### ISSUE TITLE
Mac OS X Speech tool for text-to-speech fails after first use

### ISSUE THREAD

**Issue by ivanistheone** [microsoft/vscode#118995]
- VS Code Version: 1.54.2
- OS Version: mac OS 10.13.6

Default mac os dictation dictation tool (part of Accesibility) stops working after first use.

Setup:

Steps to Reproduce:
1. On a Mac OS X computer, add a keyboard shortcut for the text-to-speech accesibility tool
<img width="669" alt="Screen Shot 2021-03-15 at 12 48 10 PM" src="https://user-images.githubusercontent.com/163966/111189726-c4498400-858c-11eb-989e-bba76b1d0c28.png">

check this doc for more info about [using built-in mac text-to-speech](https://docs.google.com/document/d/1mApa60zJA8rgEm6T6GF0yIem8qpMmnaBFYOgV32gdMc/edit).

2. Open vscode on any project that contains text (e.g. a markdown file), select a paragraph and press the keyboard shortcut defined in step 1 (CMD+OPTION+CTRL+L in my case) and you will hear the computer read the text to you.

3. Select another paragraph and press the keyboard shortcut again   
**EXPECTED**: computer to read the text again, like in step 2  
**OBSERVED**: a "terminal bell" sound is heard and nothing happens (no error printed even when running in --log=debug mode)

Does this issue occur when all extensions are disabled?: Yes

Impact:
 - For me as a user specifically, not a big deal (I'll just atom for current text editing needs)
 - Broadly for accessibility: major—users who use and depend on text-to-speech tools on Mac OS cannot currently use vscode

Happy to help debug this further in any way (just let me know how to enable more logging options so we can see what "terminal bell" means and why it is happening).

**Comment by isidorn** [comment]
@ivanistheone thanks for reporting this, I believe this is a duplicate of https://github.com/microsoft/vscode/issues/106124
which we plan to fix this milestone. Once we have a fix in insiders we will let you know

fyi @deepak1556

**Comment by ivanistheone** [comment]
Update on the issue: I recently updated to latest Mac OS X (Big Sur 11.2.3) and the text-to-speech tool is working fine now (no longer stops working after first use).

The accessibility menus have change a bit, but the option to set a "Speak selection" shortcut key is still there:
<img width="668" alt="Screen Shot 2021-04-17 at 12 58 00 PM" src="https://user-images.githubusercontent.com/163966/115120671-af418580-9f7c-11eb-9f46-82a1fefaaea9.png">

**Comment by isidorn** [comment]
@ivanistheone thanks for letting us now, so let's close this!
